packages/account-v2/src/modules/TradingAssessmentForm/TradingAssessmentForm.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { Form, Formik } from 'formik';
+import { FormSubHeader } from '../../components/FormSubHeader';
+import { RiskToleranceWarningModal } from './RiskToleranceWarningModal';
+import { TestWarningModal } from './TestWarningModal';
+import { TradingAssessmentContainer } from './TradingAssessmentContainer';
+import { getTradingAssessmentQuestions } from './tradingAssessmentQuestions';
+
+export const TradingAssessmentForm = () => {
+    const [shouldShowRiskTolerence, setShouldShowRiskTolerence] = useState(false);
+    const [shouldShowTestWarningModal, setShouldShowTestWarningModal] = useState(false);
+    const tradingAssessmentQuestions = getTradingAssessmentQuestions();
+
+    const handleAcceptRiskTolerence = () => {
```

Comment:
here you defined the function to change state, but on line 27 are changing state without it,
can we have one approach? :)

Reply:
@yauheni-deriv I did it because that we need to handle api calls for `handleAcceptRiskTolerence` i created the function just for future use, when we implement the api call. as you can see in the example image this is from `account-v1` 

![Screenshot 2024-04-05 at 10 19 26 AM](https://github.com/binary-com/deriv-app/assets/103026762/faa47405-11fe-40f5-af3a-9b82cc657726)